js/admin/turn-server.js

```diff
@@ -72,6 +73,126 @@
 			}
 		},
 
+		notifyTurnResult: function($candidates, $timeout) {
+			console.log("Received candidates", $candidates);
+			var $types = $candidates.map(function($cand) {
+				return $cand.type;
+			});
+			var $result;
+			if ($types.indexOf('relay') === -1) {
+				$result = t('spreed', 'TURN candidate generation failed, please check the settings.');
+			} else {
+				$result = t('spreed', 'The TURN server settings are valid.');
+			}
+			OC.Notification.showTemporary($result);
```

Comment:
maybe we can modify the icon to icon-loading, icon-error and icon-checkmark instead of sliding in the notifications bar